readthedocs/builds/models.py

```diff
@@ -163,7 +163,7 @@ def save(self, *args, **kwargs):  # pylint: disable=arguments-differ
     def delete(self, *args, **kwargs):  # pylint: disable=arguments-differ
         from readthedocs.projects import tasks
         log.info('Removing files for version %s', self.slug)
-        tasks.clear_artifacts.delay(version_pk=self.pk)
+        broadcast(type='app', task=tasks.clear_artifacts, args=[self.pk])
         broadcast(type='app', task=tasks.symlink_project, args=[self.project.pk])
```

Comment:
If we were using a newer version of celery, this would be a good case for chaining the tasks:
http://docs.celeryproject.org/en/latest/userguide/canvas.html#chains

I'd like to use more celery features, if we can get around to upgrading celery at some point.